scripts/images/rancher-images.txt

```diff
@@ -29,3 +29,6 @@ docker.io/rancher/mirrored-kube-logging-logging-operator:4.4.0
 docker.io/rancher/mirrored-fluent-fluent-bit:2.2.0
 docker.io/rancher/mirrored-jimmidyson-configmap-reload:v0.4.0
 docker.io/rancher/mirrored-cluster-api-controller:v1.8.3
+ghcr.io/kube-logging/fluentd:v1.15-ruby3
```

Comment:
Why do we use upstream images instead of Rancher mirrored images?

Reply:
The image tags were set by the current cluster, e.g. v1.4.3, which has a bug to get the expected tags (it also exists on v1.4.2 and booked as known issue), and the logging fallback to the default images:  those from public registry.

To fix the bug (and some others), more changes are needed, refer: https://github.com/harvester/harvester/issues/8052#issuecomment-2834730146 . To reduce the change and validation scope, those default images are added to ISO, which ensures v1.4.3->v1.5.0 upgrade does not need manually intervention.